### PR TITLE
Hardcode missing timestamps

### PR DESCRIPTION
**Pull request by MichielVieijra**
Fix manual labeling of missing timestamps

### CODE REVIEW

File: tests/push_test/breaking_test.py [hunk 1]
```diff
@@ -34,7 +28,7 @@
 
 #####################################################################
 # Set settings for QC
-minimal_gapsize = 40    #gaps defined as n times the highest frequency on IO.
+minimal_gapsize = 10    #gaps defined as n times the highest frequency on IO.
```
Comment (MichielVieijra) [inline tests/push_test/breaking_test.py:31]: To check if everything still works when gap is changed in testdata script

File: tests/push_test/breaking_test.py [hunk 2]
```diff
@@ -150,28 +146,22 @@
 from datetime import datetime
 import pandas as pd
 
-manual_missing_gaps = [{'name': 'Fictional', 'start_gap': datetime(2020,9,14,1,30), 'end_gap': datetime(2020,9,14,23,55)}] #UPDATE MANUALLY !!!!!!!!!!
+manual_missing_gaps = [{'name': 'Fictional', 'start_gap': datetime(2020,9,14,22,30), 'end_gap': datetime(2020,9,14,23,55)}] #UPDATE MANUALLY !!!!!!!!!!
```
Comment (MichielVieijra) [inline tests/push_test/breaking_test.py:149]: To check if everything still works when gap is changed in testdata script

File: tests/push_test/breaking_test.py [hunk 3]
```diff
@@ -150,28 +146,22 @@
...
-
 
 
 if not tlk_gapsdf.equals(man_gapsdf):
     print(f'ERROR: wrong gaps detection')
     
-    print(f'differences tlkit --> manual: { tlk_gapsdf.difference(man_gapsdf)}')
-    print(f'differences manual --> tlkit: {man_gapsdf.difference(tlk_gapsdf)}')
+    print(f'differences tlkit --> manual: {tlk_gapsdf[~tlk_gapsdf.apply(tuple,1).isin(man_gapsdf.apply(tuple,1))]}')
+    print(f'differences manual --> tlkit: {man_gapsdf[~man_gapsdf.apply(tuple,1).isin(tlk_gapsdf.apply(tuple,1))]}')
```
Comment (MichielVieijra) [inline tests/push_test/breaking_test.py:164]: Because there is no 'difference' function for pd.DataFrame

File: tests/push_test/breaking_test.py [hunk 4]
```diff
@@ -190,21 +180,46 @@
...
+                     ('Fictional', datetime(2020,9,15,3,10)), ('Fictional', datetime(2020,9,15,3,15)), ('Fictional', datetime(2020,9,15,3,20)),
+                     ('Fictional', datetime(2020,9,15,5,10)), ('Fictional', datetime(2020,9,15,6,45)), ('Fictional', datetime(2020,9,15,7,40)),
+                     ('Fictional', datetime(2020,9,15,12,40)), ('Fictional', datetime(2020,9,15,17,35)), ('Fictional', datetime(2020,9,15,20,40)),
+                     ('Fictional', datetime(2020,9,15,23,50)), ('1', datetime(2020,9,16,21,30))], columns=['name', 'datetime'])
+
+man_missing_timestamps_idx = pd.MultiIndex.from_frame(man_missing_timestamps_df).sort_values()
+
 tlk_missing_series = dataset.missing_obs.series
+tlk_missing_series = tlk_missing_series.reset_index().rename(columns={'index': 'name', 0: 'datetime'})
+tlk_missing_timestamps_idx = pd.MultiIndex.from_frame(tlk_missing_series).sort_values()
```
Comment (MichielVieijra) [inline tests/push_test/breaking_test.py:196]: Manually create a dataframe with all missing timestamps for the testdata script. Then create multi-index objects for the manual missing timestamps dataframe and the missing timestamps dataframe calculated by the toolkit. This in order to be consistent with the check for equality of the QC labels

File: tests/push_test/breaking_test.py [hunk 5]
```diff
@@ -190,21 +180,46 @@
...
+if not tlk_missing_timestamps_idx.equals(man_missing_timestamps_idx):
+    print(f'ERROR: wrong missing timestamps detection')
     
-print('OK!')
+    print(f'differences tlkit --> manual: { tlk_missing_timestamps_idx.difference(man_missing_timestamps_idx)}')
+    print(f'differences manual --> tlkit: {man_missing_timestamps_idx.difference(tlk_missing_timestamps_idx)}')
+    sys.exit(1)
+
+else:
+    print('OK!')
```
Comment (MichielVieijra) [inline tests/push_test/breaking_test.py:209]: Print the differences in missing timestamps between toolkit and manual labeling, consistent with check for equality of QC labels

File: tests/push_test/breaking_test.py (hunk 1 above)
Comment (vergauwenthomas) [reply tests/push_test/breaking_test.py:31]: Is the manual labeling consistent with this definition?

File: tests/push_test/breaking_test.py [hunk 6]
```diff
@@ -182,21 +180,46 @@
...
+
+#for station in tlk_missing_series.index.unique():
```
Comment (vergauwenthomas) [inline tests/push_test/breaking_test.py:211]: drop these lines

File: vlinder_toolkit/__init__.py [hunk 7]
```diff
@@ -27,20 +27,14 @@
...
-from .settings import Settings
-
-
-#Import QC checks
-from . import qc_checks
-
 
 
+from vlinder_toolkit.settings import Settings
+from vlinder_toolkit.dataset import Dataset
```
Comment (vergauwenthomas) [inline vlinder_toolkit/__init__.py:37]: @MichielVieijra Did you modify this?

File: tests/push_test/breaking_test.py (hunk 1 above)
Comment (MichielVieijra) [reply tests/push_test/breaking_test.py:31]: Yes it is

**Comment by vergauwenthomas** [comment]
merged in #89

**Pull request closed without merging**